src/components/PostDetails/PostDetails.scss

```diff
@@ -7,6 +7,10 @@
     text-align: right;
   }
 
+  &__visiblelist {
```

Comment:
according to the style of writing this file, it is better to use  `&__visible-list`